lms/djangoapps/teams/models.py

```diff
@@ -23,13 +23,13 @@ class CourseTeam(models.Model):
     course_id = CourseKeyField(max_length=255, db_index=True)
     topic_id = models.CharField(max_length=255, db_index=True, blank=True)
     date_created = models.DateTimeField(auto_now_add=True)
-    # last_activity is computed through a query
     description = models.CharField(max_length=300)
     country = CountryField(blank=True)
     language = LanguageField(
         blank=True,
         help_text=ugettext_lazy("Optional language the team uses as ISO 639-1 code."),
     )
+    last_activity_at = models.DateTimeField(auto_now_add=True, editable=True)
```

Comment:
Actually, I went poking around and found evidence that this is not going to work the way we expect, as even if you set `editable` to be `True` it will still inherit `editable=False`. I'll close this PR for right now to figure out how to fix that.